src/startupscriptgenerator/src/common/consts/ext_var_names.go

```diff
@@ -4,6 +4,7 @@ package consts
 
 const EnvironmentType string = "ORYX_ENV_TYPE"
 const ApplicationInsightsInstrumentationKeyEnvVarName string = "ORYX_AI_INSTRUMENTATION_KEY"
+const ApplicationInsightsConnectionStringEnvVarName string = "ORYX_AI_CONNECTION_STRING"
```

Comment:
There are some changes we'll also need to make in the [`logging.go`](https://github.com/microsoft/Oryx/blob/main/src/startupscriptgenerator/src/common/logging.go#L53-L62) file to actually use this connection string for the runtime images.